cnd/src/trace.rs

```diff
@@ -13,6 +13,7 @@ pub fn init_tracing(level: log::LevelFilter) -> anyhow::Result<()> {
 
     let subscriber = FmtSubscriber::builder()
         .with_max_level(level_from_level_filter(level))
+        .with_ansi(false)
```

Comment:
Would prefer if we set this according to the output of `atty`: https://github.com/softprops/atty

Reply:
Cool idea! Will do.